itests/org.openhab.core.model.script.tests/itest.bndrun

```diff
@@ -4,9 +4,8 @@ Bundle-SymbolicName: ${project.artifactId}
 Fragment-Host: org.openhab.core.model.script
 
 -runblacklist: \
-    bnd.identity;id='org.osgi.service.cm',\
-    osgi.identity;filter:='(osgi.identity=org.apache.tomcat.util.descriptor.web)'
-#
+	bnd.identity;id='org.osgi.service.cm',\
+	osgi.identity;filter:='(osgi.identity=org.apache.tomcat.util.descriptor.web)'
```

Comment:
same here

Reply:
I only changed the indentation from spaces to tabs. :slightly_smiling_face: